OneCommunityGlobal/HighestGoodNetworkApp
# Ramakrishna fix openingteams newtab

# Description
![image](https://github.com/user-attachments/assets/109b364f-9a54-4437-8503-528e1eaf59c8)


## Related PRS (if any):
Please use latest  backend development branch to test this changes
…

## Main changes explained:
- Updated the TeamReport.jsx with an api call to get teamdetails 
- Updated the loader spinner in TeamReportLogs.jsx
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin/owner user
5.  Go to  Reports -> Teams
6.  Click an one of the team Name and navigate their component  and navigate back to page and try to navigate to another team and check if each time you navigate to another team respective team details are rendered or not.
7. Now instead of clicking on team name now try to open the team name in new tab like for different team with different team and check each time the respective  team get rendered 
8. For reference please watch uploaded video.
## video reference for testing


https://github.com/user-attachments/assets/2f51dbce-0d8e-47a7-a445-a09980bd5baa




## Note:
When I was about to raise the pull request, I received some new updates from the development branch. These updates resulted in differences in the team report, particularly affecting the rendering of team members. I'm uncertain whether this behavior is an issue or expected. However, it's important to note that the team members not rendering is unrelated to this specific pull request.
![image](https://github.com/user-attachments/assets/874f46e2-db14-4dfd-bd1a-3b8502b33db1)
